new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -579,12 +716,74 @@ public static boolean patchDomainCustomResource(String domainUID, String namespa
    * @return List of Domain Custom Resources
    */
   public static DomainList listDomains(String namespace) {
-    KubernetesApiResponse<DomainList> response = crdClient.list(namespace);
+    KubernetesApiResponse<DomainList> response = null;
+    try {
+      response = crdClient.list(namespace);
+    } catch (Exception ex) {
+      logger.warning(ex.getMessage());
+      throw ex;
+    }
     return response != null ? response.getObject() : new DomainList();
   }
 
-  // --------------------------- config map ---------------------------
 
+  /**
+   * Get Domain Custom Resource objects in all namespaces.
+   *
+   * @return Object Domain Custom Resources object
+   */
+  public static Object getDomainObjects() {
```

Comment:
I'm confused.  Why do we still have these methods?

Reply:
Removed it.